### DISCUSSION TITLE
Place to publish?

### DISCUSSION THREAD

**Discussion by aiqc** [quarto-dev/quarto-cli#2179] [Q&A] [answered]
Is there someplace I can publish my Quarto documents publicly without having to host my own server?

**Comment by mcanouil** [comment, marked as answer]
Yes, it's called <https://quartopub.com/> and it's free.
And here the Quarto documentation on how to get your documents on that platform: <https://quarto.org/docs/publishing/quarto-pub.html>.

PS: can you change the Discussion category from "General" to "Q&A" (and select the answer if there is one)? 😉
